src/pages/SkillSharePage/components/Steps/Skills/Skills.tsx

```diff
@@ -0,0 +1,83 @@
+import Button from "@/src/components/SeButton/SeButton";
+import Tag from "@/src/components/Tag/Tag";
+import { FormikProps, FormikValues } from "formik";
+import { useState } from "react";
+
+type SkillsStepProps = {
+  form: FormikProps<FormikValues>;
+  name: "Skills";
+};
+
+function SkillsStep({ form }: SkillsStepProps) {
+  const [formSkills, setFormSkills] = useState<string[]>([])
+  const [skillItem, setSkillItem] = useState('');  
```

Comment:
I would change this to skillItem input for better readability. This name almost seems like an array when it is just the `skillItemInputField`